core/src/main/java/hudson/model/View.java

```diff
@@ -1268,6 +1277,9 @@ public static View create(StaplerRequest req, StaplerResponse rsp, ViewGroup own
         if (mode==null || mode.length()==0) {
             if(isXmlSubmission) {
                 View v = createViewFromXML(name, req.getInputStream());
+                if (!owner.getACL().hasCreatePermission(Jenkins.getAuthentication(), owner, v.getDescriptor())) {
+                    throw new Failure(Messages.View_CreatePermissionMissing(v.getDescriptor().getDisplayName()));
```

Comment:
`AccessDeniedException`, or better yet define a new `ACL.checkCreatePermission` overload for this.